core/controllers/admin.py

```diff
@@ -831,18 +831,112 @@ def _generate_dummy_classroom(self) -> None:
             skill_id_5 = skill_services.get_new_skill_id()
             skill_id_6 = skill_services.get_new_skill_id()
 
+            question_id_1 = question_services.get_new_question_id()
```

Comment:
The changes in this file are used to generate the dummy classrooms with data.

Reply:
Ack. Given these changes, though, shouldn't the admin page text be updated with an explanation of what the generated classroom will contain?

Reply:
Updated the text in the admin config tab.
Text: 
`Generate a dummy math classroom with 5 published topics containing 1 skill per topic and 3 questions linked per skill.`